### PR TITLE
fix error in claim_list_mine

### PR DESCRIPTION
**Pull request by tau3**

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks for following up on this @tau3 !

**Comment by eukreign** [comment]
@nikooo777 does this solve the problem?

**Comment by nikooo777** [comment]
I can test the case that broke the daemon but in regards of the code It's up to either your or @jackrobison to approve.

I'll let you know ASAP

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ at anytime.
   * download blockchain headers from s3 before starting the wallet when the local height is more than `s3_headers_depth` (a config setting) blocks behind
   * track successful reflector uploads in sqlite to minimize how many streams are attempted by auto re-reflect
   * increase the default `auto_re_reflect_interval` to a day
+  * `claim_list` and `claim_list_mine` in Daemon `return` sorted results
```
Comment (jackrobison) [inline CHANGELOG.md]: "return" doesn't need code tags

File: lbrynet/daemon/Daemon.py [hunk 2]
```diff
@@ -163,6 +163,11 @@ def __call__(self):
         return d
 
 
+def sort_claim_results(claims):
+    claims.sort(key=lambda d: (d['height'], d['name'], d['claim_id'], d['txid'], d['nout']))
+    return claims
```
Comment (jackrobison) [inline lbrynet/daemon/Daemon.py]: This doesn't need to return, the provided `claims` list is sorted in place.

Comment (eukreign) [reply lbrynet/daemon/Daemon.py]: `sort_claim_results` is provided as a Deferred callback so if it doesn't return then the subsequent callbacks will not get the claims.

**Comment by eukreign** [comment]
@tau3 looks like you just need to make the fix to the CHANGELOG per @jackrobison comment

@jackrobison other than the CHANGELOG does the implementation look correct to you?

File: lbrynet/daemon/Daemon.py (hunk 2 above)
Comment (jackrobison) [reply lbrynet/daemon/Daemon.py]: Ok, makes sense

**Comment by tzarebczan** [comment]
Yep, let's merge and I'll open a separate issue for allowing sorting arguments, similar to what we did recently for file_list.

**Comment by tzarebczan** [comment]
Thanks for patching this us @tau3! Same as with the last PR, please see https://lbry.io/faq/tips if you are interested in a tip for this contribution :)

**Pull request closed without merging**